src/api.js

```diff
@@ -28,7 +28,18 @@ export const giveVotes = (articleId, upDownVote) => {
 	} else {
 		upDownVote = -1;
 	}
+	return newsAPI.patch(`/articles/${articleId}`, {
+		inc_votes: `${upDownVote}`,
+	});
+};
+
+export const postComment = (articleId, comment) => {
 	return newsAPI
-		.patch(`/articles/${articleId}`, { inc_votes: `${upDownVote}` })
-		.then((response) => {});
+		.post(`/articles/${articleId}/comments`, {
+			username: "grumpy19",
+			body: comment,
+		})
+		.then((response) => {
```

Comment:
This .then() block is redundant as it isn't processing the response in anyway - the promise we return if we just return newsAPI.patch() will resolve to the response anyway so us returning the response inside the .then() is not adding anything to the code. If you ever find yourself purely just returning the response from the .then() without processing it in anyway, that tells you you don't need it! :)